### DISCUSSION TITLE
Notes der Platz-Reservationen für alle anzeigen

### DISCUSSION THREAD

**Discussion by michelroger78** [tkrebs/ep3-bs#692] [General]
Wir hätten gerne die Bemerkungen für die Platzreservation für alle angezeigt. Ich habe das versucht in module\Square\view\square\square\index.phtml einzubauen. Habe das aber nicht geschafft.
Hat jemand eine Idee wie das gemacht werden könnte?

**Comment by phuber-asak** [comment]
Hallo, hast du das irgendwie hinbekommen?

**Comment by tkrebs** [comment]
Ohne Test und Gewähr:

https://github.com/tkrebs/ep3-bs/blob/master/module/Square/view/square/square/index.occupied.phtml

Am Ende ergänzen um:

```php
foreach ($this->bookings as $booking) {
    echo nl2br($booking->getMeta('notes'));
}
```

Wahlweise mit oder ohne Prüfung auf angemeldeten Benutzer.
